src/main.rs

```diff
@@ -957,4 +986,93 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    fn test_virtio_console() {
+        test_block!(tb, "", {
+            let guest = Guest::new();
+            let mut child = Command::new("target/debug/cloud-hypervisor")
+                .args(&["--cpus", "1"])
+                .args(&["--memory", "size=512M"])
+                .args(&["--kernel", guest.fw_path.as_str()])
+                .args(&["--disk", guest.disks[0].as_str(), guest.disks[1].as_str()])
+                .args(&[
+                    "--net",
+                    "tap=,mac=12:34:56:78:90:ab,ip=192.168.2.1,mask=255.255.255.0",
+                ])
+                .args(&["--console", "tty"])
+                .stdout(Stdio::piped())
+                .stderr(Stdio::piped())
+                .spawn()
+                .unwrap();
+
+            thread::sleep(std::time::Duration::new(10, 0));
```

Comment:
In the other tests this has been bumped to 20.

Reply:
It is changed accordingly.